app/assets/javascripts/external_activities/matedit.js

```diff
@@ -1,27 +1,31 @@
 function get_edit_resource_popup(external_activity_id, popup_options){
   var lightboxConfig = {
         content:"<div style='padding:10px'>Loading...Please Wait.</div>",
-        title:"Edit External Activity"
+        title:"Edit Options"
     };
     list_lightbox=new Lightbox(lightboxConfig);
-    jQuery(".n.move_handle").css({"width":"400px"});
     jQuery(".buttons")[0].innerHTML = jQuery(".buttons")[0].innerHTML + "<a href='javascript:void(0)' style='float:right;margin-top: 5px;margin-right: 10px' class='button' onclick='close_popup()'>Cancel</a>"
     var target_url = "/eresources/999/edit";
     if (popup_options && popup_options.use_short_form) {
       target_url = "/eresources/999/edit_basic"
     }
     target_url = target_url.replace('999',external_activity_id);
```

Comment:
It was due to changes for Docker precompile = https://github.com/concord-consortium/rigse/commit/67b4dd7dc2b3e97d32112edac2ea14101a8de19e